OpenNefia.Core/Input/Client/EngineContexts.cs

```diff
@@ -37,6 +37,9 @@ public static void SetupContexts(IInputContextContainer contexts)
 
             common.AddFunction(EngineKeyFunctions.UINextPage);
             common.AddFunction(EngineKeyFunctions.UIPreviousPage);
+            common.AddFunction(EngineKeyFunctions.UINextTab);
+            common.AddFunction(EngineKeyFunctions.UIPreviousTab);
+            common.AddFunction(EngineKeyFunctions.UIBacklog);
```

Comment:
This shouldn't go in the root context, only in `"field"`.

Reply:
doing that makes the UiGroups unresponsive

Reply:
talked about it, changed in 9eb449f